client/src/app/app.component.scss

```diff
@@ -1,3 +1,9 @@
-nav {
-  background-color: #222;
+// Required for bs4
+@import '~bootstrap/scss/functions';
```

Comment:
You can add these to `src/styles.scss` instead of the app component scss

Reply:
fixed! good catch. 👍